packages/pluggableWidgets/gallery-native/README.md

```diff
@@ -0,0 +1,90 @@
+
+### 2.3 Items
+
+In the items group you can define the properties related to the items being presented.
+
+### 2.3.1 Page size
+
+Defines the amount of data shown for each page or the limit to be presented.
+
+### 2.3.2 Pagination
+
+You can choose between `Load more button` (button being presented below) or `Virtual scrolling` (mechanism that automatically loads data when the users reaches the bottom of the scrollbar).
+
+### 2.3.3 Empty Message
+
+Here you can place widgets that will be presented when no items are available to be shown:
+
+![EmptyMessage](./assets/custom-empty-message.png)
+
+### 2.3.4 Dynamic Item Class
+
+In the new Gallery widget we offer an option to dynamically apply a style object to a specific item. You can achieve this by adding an expression based on the item values (attributes) like the example below.
+
+In this example we check the value of `CountryName`, then if the attribute is equal to `Spain` apply the style object `myCustomStyle` to the items:
+
+![DynamicItemClass](./assets/dynamic-item-class.png)
+
+## 2.4 Events
+
+The new Gallery widget can trigger some events while iterating with it.
+
+### 2.4.1 On Click Action
+
+Triggers an action (such as a nanoflow, Show page, etc. action) when the end-user clicks in one of the items.
+
+### 2.4.2 On Pull Down Action
+
+Triggers an action (such as a nanoflow, Synchronize, etc. action) when the end-user swipe down of the list.
+
+### Filtering
+
+In order to enable filtering within the gallery items you need to select the desired attributes to be filtered in the Filtering tab.
+
+The attributes selected here will be used for the matching filter placed inside the composable region. When using multiple attributes, the filters will automatically select the matching attributes and then compose the desired filter value in an OR expression. Make sure you just have one filter widget for each type, for example Text Filter and Number Filter.
+
+> ⚠️ **If a filter is being used and its type does not match with any selected attribute it will throw an error requesting you to select the correct filter widget.**
+
+An example of filtering configuration:
+
+![FilteringConfiguration](./assets/filtering-configuration.png)
+
+An example of text filter:
+
+![TextFilter](./assets/text-filter.png)
+
+## Custom styling
+
+The look and feel of the widget is fully customizable. Check out the <a href="https://docs.mendix.com/refguide/mobile/designing-mobile-user-interfaces/widget-styling-guide/#1132-gallery">Native Mobile Styling documentation</a> for a full overview of the available style customizations.
+
+## Development and contribution
+
+1. Install NPM package dependencies by using: `npm install`. If you use NPM v7.x.x, which can be checked by executing `npm -v`, execute: `npm install --legacy-peer-deps`.
```

Comment:
Still uses npm

Reply:
Maybe the `Development and contribution` section can be removed, there is no `development and contribution` section in the other widgets.

Reply:
Agree